### PR TITLE
Integrating the Yi series models

### PR DESCRIPTION
**Pull request by Haijian06**
Integrating the Yi series models.We have added several files that utilize the Yi models.

Tested (run the relevant ones):

- [x] Code formatting: `bash format.sh`
- [x] Any manual or new tests for this PR (please specify below)

### CODE REVIEW

**Comment by Michaelvll** [comment]
Hi @Haijian06, thanks a lot for adding the integration for Yi series models! We are excited to see the new support of those model. Could we add a brief README file with instructions for launching the YAMLs and calling the endpoints? It would be very appealing to show a GIF about the model's capability, e.g., how well YiCoder can do with coding tasks. : )

Please let me know if you would like to do it. Otherwise, we can get this PR in first.

**Comment by Haijian06** [comment]
Thanks to @Michaelvll and the team for creating this framework. I tried it out early on, and the experience has been truly fantastic! Can we get this PR in first? I will continue improving the experience of using the Yi model in Skypilot next. Once again, thank you to @Michaelvll and the team for your hard work.

File: llm/yi/yi15-34b.yaml
Comment (Michaelvll) [inline llm/yi/yi15-34b.yaml:16]: If we are not adding instructions for `sky serve up`, we can leave these sections out.

**Review by Michaelvll — APPROVED** [review]
Thanks for adding the example for Yi models @Haijian06! LGTM! Let's improve the readme after this PR is merged. Great to see this happens.

File: llm/yi/yi15-34b.yaml
Comment (Haijian06) [reply llm/yi/yi15-34b.yaml:16]: Okay. Thanks.

Comment (Haijian06) [reply llm/yi/yi15-34b.yaml:16]: Hi @Michaelvll  am I to remove this section from all yaml files? Does it matter if I leave the service section here first, maybe we will add it later when we optimize the documentation.

Comment (Michaelvll) [reply llm/yi/yi15-34b.yaml:16]: Optimizing the docs later sounds good to me. Merging for now.

Comment (Haijian06) [reply llm/yi/yi15-34b.yaml]: @Michaelvll Thanks a lot, I've changed the file!

**Pull request merged**
